CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 ## Unreleased
 
+### Added
+
+- Added new functions to database module
+  -`database.SetDefaultValuesFromItem(accessName, itemName, item)` 
```

Comment:
Is this the best place for this function? Because the scope sounds like a generic database scope and this is very item specific

Reply:
Well an item is not just equipment it also means an object of a table, therefore it is theoretically generalized, even if I only yet used it for equipment/items.

Reply:
ahhh, my bad, I thought of equipment items. Nevermind then!